frontend/src/methods/key.ts

```diff
@@ -257,7 +274,11 @@ const registerInterceptors = () => {
     request: async (url, config?: { headers?: Headers; method?: string }) => {
       url = encodeURI(url)
 
-      if (!/^\/(api|image)/.test(url) || url.indexOf('/api/auth.') !== -1) {
+      if (
+        !/^\/(api|image)/.test(url) ||
+        (url.indexOf('/api/auth.') !== -1 &&
+          url.indexOf('/api/auth.AuthService/RevokePublicKey') === -1)
```

Comment:
if its in the middle of the URL
```suggestion
        !url.includes('/api/auth.')
```

if its at the start of the URL
```suggestion
        !url.startsWith('/api/auth.')
```

but i guess if its at the start wouldnt `/^\/(api|image)/` already cover it?

Reply:
we need to add signature to all requests that start with `/api` and `/image`, but not including:
- `/api/auth.AuthService/ConfirmPublicKey`
- `/api/auth.AuthService/RegisterPublicKey`

I agree that it would be nice to write it cleaner, but the condition you suggest is different.

Reply:
Another option to write this is this:
```ts
      if (!url.startsWith('/api') || !url.startsWith('/image')) {
        return [url, config]
      }

      if (url.startsWith('/api/auth.AuthService/') && !url.startsWith('/api/auth.AuthService/RevokePublicKey')) {
        return [url, config]
      }
```

Reply:
Used `startsWith` where possible